README.md

```diff
@@ -16,14 +16,29 @@ yarn add @rehooks/online-status
 ## Usage
 
 ```js
-import useOnlineStatus from '@rehooks/online-status';
+import useOnlineStatus from "@rehooks/online-status";
 
 function MyComponent() {
-  let onlineStatus = useOnlineStatus();
+  let onlineStatus = useOnlineStatus({
+    onStatusChange: online =>
```

Comment:
I feel like this should be implemented as something like `<OnlineToaster status={onlineStatus} />` that triggers a toast on prop change rather than introducing a new concept to Hooks

Reply:
This isn't really a new concept for hooks `useImperativeMethods` is basically this.

I think it's fine, but I would assume most of the time you'd use it for `setX` calls

Reply:
@fouad I can see the point you are making here. I added those hooks because I sometimes need to call some function whenever the `online` status changes. What do you people think? Should this be implemented here or not? 🙏